<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SR_settingqtwidgetsclass</class>
 <widget class="QWidget" name="SR_settingqtwidgetsclass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>超分辨率引擎1</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>没有模型文件可以通过下面自动下载来获取模型文件

推理模型简介:
realesr-animevideov3-x2 — 适用于动漫的图片和视频
realesr-animevideov3-x3 — 适用于动漫的图片和视频
realesr-animevideov3-x4 — 适用于动漫的图片和视频
realesrgan-x4plus-anime — 适用于动漫的图片和视频
realesrgan-x4plus — 适用于正常照片和视频
感谢开源项目: https://github.com/xinntao/Real-ESRGAN 对本引擎提供的帮助和参考</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="elideMode">
          <enum>Qt::ElideLeft</enum>
         </property>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>自动下载模型</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>安装教程：下载模型 -&gt; 刷新 -&gt; 加载模型</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="DownloadListWidgetsClass" name="DownloadlistWidget"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>手动下载模型</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>34</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="to_web_d_pushButton">
             <property name="text">
              <string>跳转网页下载</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>使用教程:
1,到网页下载模型.
2,复制模型文件到模型储存位置(文件名称必须为models)
3,选择下载的模型 
4,验证模型是否可用</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>189</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="1">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>35</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>151</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>模型存储位置：</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>5</number>
            </property>
            <item>
             <widget class="QLabel" name="model_save_path_label">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="model_save_path_pushButton">
              <property name="text">
               <string>选择</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="show_model_path_pushButton">
              <property name="text">
               <string>显示存储位置</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="model_path_help_pushButton">
              <property name="toolTip">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;必须放在名称为&lt;span style=&quot; font-weight:700; color:#ff2600;&quot;&gt;models&lt;/span&gt;的文件夹下&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../Resource.qrc">
                <normaloff>:/new/prefix1/Resource/image/winOCRUI/questionmark.circle.svg</normaloff>:/new/prefix1/Resource/image/winOCRUI/questionmark.circle.svg</iconset>
              </property>
              <property name="autoRepeatDelay">
               <number>300</number>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>模型设置:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>默认推理模型:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="model_Instrumental_selection_comboBox"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="model_up_pushButton">
              <property name="text">
               <string>刷新</string>
              </property>
              <property name="icon">
               <iconset resource="../Resource.qrc">
                <normaloff>:/new/1_1image/Resource/image/winOCRUI/1.1/arrow.triangle.2.circlepath.svg</normaloff>:/new/1_1image/Resource/image/winOCRUI/1.1/arrow.triangle.2.circlepath.svg</iconset>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>23</width>
                <height>23</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="verify_pushButton">
              <property name="text">
               <string>验证是否可用</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>模型放大倍数:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="scaled_label">
              <property name="text">
               <string>未知</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>视频默认输出文件格式:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="video_radioButton_default">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>跟随输入</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">out_video_buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="video_radioButton_mp4">
              <property name="text">
               <string>mp4</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">out_video_buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="video_radioButton_mov">
              <property name="text">
               <string>mov</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">out_video_buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>图片默认输出类型:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="image_radioButton_png">
              <property name="text">
               <string>PNG</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">out_image_buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="image_radioButton_jpg">
              <property name="text">
               <string>JPG</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">out_image_buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DownloadListWidgetsClass</class>
   <extends>QListWidget</extends>
   <header>DownloadListWidgetsClass.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="out_video_buttonGroup"/>
  <buttongroup name="out_image_buttonGroup"/>
 </buttongroups>
</ui>
